admin-dev/filemanager/execute.php

```diff
@@ -31,12 +33,7 @@
 require_once $language_file;
 
 $base = $current_path;
-
-if (isset($_POST['path'])) {
-    $path = $current_path.str_replace("\0", "", $_POST['path']);
-} else {
-    $path = $current_path;
-}
+$path = isset($_POST['path']) ? ($current_path . $_POST['path']) : $current_path;
```

Comment:
With your PR, `isset($_POST['path'])` is now always true.

Reply:
There are many require above, I can't trust anything, that's why I keep the isset test :/